spec/features/scheduling/change_proposed_schedule_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.feature "Change proposed schedule" do
+  scenario do
+    given_there_is_an_edition_with_a_proposed_schedule
+    when_i_visit_the_summary_page
+    and_i_click_on_change_date
+    then_i_see_the_current_date_and_time_selected
+
+    when_i_set_a_new_proposed_schedule_datetime
+    then_i_see_the_new_proposed_schedule_datetime
+  end
+
+  def given_there_is_an_edition_with_a_proposed_schedule
+    @current_datetime = Time.current.tomorrow.change(hour: 10)
```

Comment:
minor, but how would you feel about avoiding `current` here and instead something like `proposed_time`. Just that I read current as now.

Reply:
Also apologies, I've noticed that you didn't introduce this so don't worry if it's a pain.

Reply:
It didn't bother me particularly, but I've gone with just plain 'datetime', which hopefully stops people making any assumptions about it ;-).